DistroLauncher-Tests/DistroLauncher-Tests/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include "stdafx.h"
+#include <cstdio>
+#include "gtest/gtest.h"
```

Comment:
How is gtest pulled? We just expect to have the header installed on the machine (which is ok IMHO) and the `GTest/` subfolder will just compile/link against it?

Reply:
Ack, thanks for the explanation! :)